### PR TITLE
Upgrade Compose project's AGP to alpha02.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

File: compose/.idea/misc.xml [hunk 1]
```diff
@@ -13,10 +13,11 @@
       <item index="3" class="java.lang.String" itemvalue="org.openjdk.jmh.annotations.BenchmarkMode" />
     </list>
   </component>
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
```
Comment (maybeYouShouldBeTheLawyer) [inline compose/.idea/misc.xml:16]: Is this required by the bump / worth doing in this PR? 

Also – for my own education – what is this? A Google search + search in #compose were surprisingly light on details.

**Review by maybeYouShouldBeTheLawyer — APPROVED** [review]
Questions because I don't know stuff. Otherwise lgtm.

**Review by steve-the-edwards — APPROVED** [review]

File: compose/.idea/misc.xml (hunk 1 above)
Comment (zach-klippenstein) [reply compose/.idea/misc.xml:16]: Idk, intellij man. It's been a while since we've updated this, so I'd rather just commit it than continue fighting it.

Comment (zach-klippenstein) [reply compose/.idea/misc.xml:16]: https://github.com/square/workflow-kotlin/pull/438

**Pull request merged**